src/WP_CLI/CommandWithMeta.php

```diff
@@ -142,6 +142,9 @@ function ( $a, $b ) use ( $orderby, $order ) {
 	 * <key>
 	 * : The name of the meta field to get.
 	 *
+	 * [--single]
+	 * : Whether to return a single value. This parameter has no effect if $key is not specified.
```

Comment:
<key> is required here in this command. I think command wont even run without that argument. Not sure second sentence is needed here. What do you think?

Reply:
I've updated the documentation (Removed 2nd sentence).

I tried the command without `--single` and it is working fin without it. I thin `--` makes it optional argument. 
<img width="585" alt="image" src="https://github.com/user-attachments/assets/ae12e72c-2ea8-4518-800b-b8f8472b63d3" />